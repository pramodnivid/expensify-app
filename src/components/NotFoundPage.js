
import React from "react"
import ReactDOM from "react-dom"
import {BrowserRouter,Route,Switch,NavLink} from "react-router-dom"

const NotFoundPage=() => (
    <div>
        Not Found Page -
     </div>
)
export default NotFoundPage;
