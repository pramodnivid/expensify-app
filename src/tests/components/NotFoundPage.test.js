import React from "react"
import ReactShallowRenderer from "react-test-renderer/shallow"
import NotFoundPage from "../../components/NotFoundPage"
import {shallow} from "enzyme"
test("Not found page",()=>{
    const wrapper = shallow(<NotFoundPage />);
    //expect(wrapper.find("h1").text()).toBe("Expensify")
    expect((wrapper)).toMatchSnapshot()

})
