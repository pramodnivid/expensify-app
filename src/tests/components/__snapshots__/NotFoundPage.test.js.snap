// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found page 1`] = `
<div>
  Not Found Page -
</div>
`;
